server/commision.js

```diff
@@ -0,0 +1,395 @@
+import https from "https";
+import querystring from "querystring";
+import util from "util";
+/*
+const https = require("https");
+const querystring = require("querystring");
+const util = require("util");
+*/
+//const key = "bfc05de4947f464f85ecd85deff6895e";
+const key = "7db287c206374b2f911ddc918879983d"; // Dan's API key
+
+var auth = "939094d5c42a409bbbfa7fe9f5bc065521ecfa58a75b47d2bdb4383fd53a00e7";
+//to authenticate with MB during debugging, you need to uncomment these next few lines.
+//submit a blank auth_key to MB and use printAuthorizationKey to request a new auth_key.
+//then, take the returned new auth_key and paste it into var auth = "..." above.
+//eventually, this kludge method will be replaced by MindbodyQueries which will do this
+//more cleanly.
+//var auth = "";
+//printAuthorizationKey("Siteowner", "apitest1234");
+
+//to add client visits: sandbox -> classes -> sign in -> search for client -> add
+//to add sales: sandbox -> retail -> client search -> add item
+
+function getRequest(path, payload, callback) {
+  var options = getOptions(path);
+  options.method = "GET";
+  options.path += "?" + querystring.stringify(payload);
+  console.log("GET " + options.host + options.path);
+  const req = https.request(options, callback);
+  req.on("error", e => console.log(e));
+  req.end();
+}
+
+function postRequest(path, payload, callback) {
+  var options = getOptions(path);
+  options.method = "POST";
+  payload = JSON.stringify(payload);
+  options.headers["Content-Length"] = payload.length;
+  console.log("POST " + options.host + options.path);
+  const req = https.request(options, callback);
+  req.on("error", e => console.log(e));
+  req.write(payload);
+  req.end();
+}
+
+function getOptions(path) {
+  return {
+    host: "api.mindbodyonline.com",
+    path: "/public/v6/" + path ,
+    headers: {
+      "Api-Key": key,
+      "SiteId": "-99",
+      "Authorization": auth,
+      "Content-Type": "application/json"
+    }
+  };
+}
+
+export function printAuthorizationKey(username, password) {
+  var payload = {
+    "Username": username,
+    "Password": password
+  };
+  postRequest("usertoken/issue", payload, res => {
+    res.on("data", d => console.log(JSON.parse(d)));
+  });
+}
```

Comment:
This should all be removed and replaced with the API manager, unless you want to keep it for testing.

Reply:
The API manager is mostly not working right now. I got getStaff and getAuth working because I needed them for my part of this ticket, but we can't replace this working code with MindbodyAccess until it's ready. I would rather merge this code in now and get MBAccess working in a separate branch/ticket.